traits/src/registry.rs

```diff
@@ -18,6 +19,14 @@ pub trait Registry<AssetId, AssetName, Balance, Error> {
 	}
 }
 
+pub trait RegistryQueryForEvm<AssetId, NameStringLimit, Error> {
+	fn retrieve_asset_name(name: AssetId) -> Result<BoundedVec<u8, NameStringLimit>, Error>;
+
+	fn retrieve_asset_symbol(name: AssetId) -> Result<BoundedVec<u8, NameStringLimit>, Error>;
+
+	fn retrieve_asset_decimals(name: AssetId) -> Result<u8, Error>;
+}
+
```

Comment:
we already have traits that returns decimals - it is called InspectRegistry.. We should not really create a new trait to do the same. 

we can just extend that trait to return name and symbol too

btw.  I saw you use some pallets directly in evm impl,  so you could use pallet_registry directly as well, without the need of any traits.

Reply:
Resolved by https://github.com/galacticcouncil/HydraDX-node/pull/646/commits/9215cb78aded2bf00a51cc72d5297c4c99c115dd and following commits